### PR TITLE
bug patch 0.1

### PR DESCRIPTION
**Pull request by ashi72**
Various small bugs including PropType errors, unique key errors, and component (usu mui) attributes that are misused.

### CODE REVIEW

File: src/Components/Profile/ProfilePopup.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ function ProfilePopup({
       }
       if (data.color != null) {
         setBg(data.color);
-        console.log('background', data.color);
+        // console.log('background', data.color);
```
Comment (soliavalentine) [inline src/Components/Profile/ProfilePopup.js:137]: can we just delete the comment?

File: src/Pages/Landing.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ function Landing() {
         unapproved.push(js);
       });
     } catch (error) {
-      console.log(error);
+      // eslint-disable-next-line no-alert
```
Comment (soliavalentine) [inline src/Pages/Landing.js:28]: can we either configure an es lint to ignore alerts or alternatively just take the warning? or does it throw a console error

File: src/Services/jobseeker-service.js [hunk 3]
```diff
@@ -20,11 +20,9 @@ export const fetchJobseeker = async (email) => {
   try {
     const docSnap = await getDoc(docRef);
     if (docSnap.exists()) {
-      console.log('jobseeker: ', docSnap);
+      // console.log('jobseeker: ', docSnap);
```
Comment (soliavalentine) [inline src/Services/jobseeker-service.js]: rm comment 😀

File: src/firebase.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ import {
 // For Firebase JS SDK v7.20.0 and later, measurementId is optional
 // eslint-disable-next-line no-unused-vars
 const key = process.env.REACT_APP_FIREBASE_API_KEY;
-console.log(process.env);
+// console.log(process.env);
```
Comment (soliavalentine) [inline src/firebase.js]: same here

**Review by soliavalentine — APPROVED** [review]
yuhhh slay job aaron app is looking so fresh and so clean! I added a couple of clarifyin  questions// comments but otherwise looks good to go!

File: src/firebase.js (hunk 4 above)
Comment (ashi72) [reply src/firebase.js]: oki

File: src/Services/jobseeker-service.js (hunk 3 above)
Comment (ashi72) [reply src/Services/jobseeker-service.js]: oki

**Pull request merged**